x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -233,12 +234,24 @@ export class IndexActionsContextMenu extends Component {
                 <li key={indexName}>{indexName}</li>
               ))}
             </ul>
+            <EuiCallOut
+              title="Proceed with caution!"
+              color="warning"
+              iconType="help"
+            >
+              <p>
+                Force merging a large index or an index that is not read-only can
+                potentially cause performance and stability issues in the cluster
+                if it is not run properly (run against non-read-only indices) or run during peak hours.
+              </p>
+            </EuiCallOut>
```

Comment:
Can we add a spacer below this callout to create a gap between this and the input?

```jsx
<EuiSpacer size="s" />
```